### DISCUSSION TITLE
Apps/projects using twilio-video.js

### DISCUSSION THREAD

**Discussion by seancoleman2** [twilio/twilio-video-app-react#739] [Show and tell]
@RusseII had a great idea in #725 to start compiling a list of projects that use twilio-video.js for others to reference.

So...if you have an app/project using the Twilio Video Javascript SDK (need not be React), share the URL and/or GitHub link, and a brief description.

I'll go first with some examples from the Twilio side. 

**Twilio Video Diagnostics App**: End-user diagnostics tool
- https://github.com/twilio/twilio-video-diagnostics-react-app
- https://video-diagnostics.twilio.com/

**Twilio JS Room Monitor**: Tool for developers building with twilio-video.js 
- https://github.com/twilio/twilio-video-room-monitor.js

**Comment by RusseII** [comment]
Hi! We converted the current app to nextjs: https://github.com/alwaysbegrowing/twilio-video-app-react-nextjs 

In progress of using twilio-video to build virtual co-working space: https://github.com/alwaysbegrowing/fokus

Also an old video interviewing app that had a few features for recruiters: https://github.com/RusseII/beta.live.deephire.com (wouldn't recommend looking at this one tho as it hasn't been updated for some time)

**Comment by 1dolinski** [comment]
Hey! We use Twilio video on https://www.Vibehut.io 

Take a look and I would be happy to chat with any builders!
